sdk/py/apepay/__init__.py

```diff
@@ -429,6 +429,17 @@ def amount_left(self) -> int:
     def time_left(self) -> timedelta:
         return timedelta(seconds=self.contract.time_left(self.creator, self.stream_id))
 
+    @property
+    def total_time(self) -> timedelta:
+        info = self.info  # NOTE: Avoid calling contract twice
```

Comment:
the note is exactly why it is no so great to hide contract calls behind a property